src/types/SimulatorUI.ts

```diff
@@ -0,0 +1,14 @@
+import { ConnectorStatus } from './ConnectorStatus';
+
+export type SimulatorUI = {
+  hashId: string;
+  stationInfo: ChargingStationInfoUI;
+  connectors: Array<ConnectorStatus>;
+};
+
+export type ChargingStationInfoUI = {
+  chargePointModel: string;
+  chargePointVendor: string;
+  chargingStationId?: string;
+  firmwareVersion?: string;
+};
```

Comment:
It's more related to the internal worker messaging protocol using the BC or the builtin default messaging.